source/base/polynomials_abf.cc

```diff
@@ -47,7 +48,8 @@ namespace
 
 template <int dim>
 PolynomialsABF<dim>::PolynomialsABF(const unsigned int k)
-  : my_degree(k)
+  : TensorPolynomialsBase<dim>(k, compute_n_pols(k))
+  , my_degree(k)
   , polynomial_space(get_abf_polynomials<dim>(k))
   , n_pols(compute_n_pols(k))
```

Comment:
This suggests that you now have duplicate data in the base class and the derived class. You could get rid of the `n_pols` variable by writing `this->n()` everywhere. I suspect that the same is true for `my_degree`.

Reply:
This one works!